plugin/storage/es/options.go

```diff
@@ -214,6 +216,10 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		"(experimental) Use read and write aliases for indices. Use this option with Elasticsearch rollover "+
 			"API. It requires an external component to create aliases before startup and then performing its management. "+
 			"Note that "+nsConfig.namespace+suffixMaxSpanAge+" is not taken into the account and has to be substituted by external component managing read alias.")
+	flagSet.Bool(
```

Comment:
@yardbirdsax I have added boolean flag `--es.create-index-templates`  which creates templates at startup when enabled. By default it is enabled.

Reply:
That's awesome @pavolloffay , thanks!